### DISCUSSION TITLE
IPv6 loopback

### DISCUSSION THREAD

**Discussion by notEvil** [nextcloud/all-in-one#4891] [Questions] [answered]
Hi,

two days ago I removed the A records to my server in preparation for DS-lite and everything works over IPv6 except the self check (administration -> overview). It lists ~10 entries all of which state "To allow this check to run you have to make sure that your webserver can connect to itself. Therefor it must be able to resolve and connect to at least one its `trusted_domains` or the `overwrite.cli.url`". I remember that for IPv4 loopback I had to add the domain name as exception to the rebind protection. But for IPv6 the router doesn't seem to care and so my question is: what needs to change to allow loopback requests if the router doesn't cooperate?

**Comment by notEvil** [comment]
Update: the router and rebind protection is irrelevant. `curl` from nextcloud-aio-nextcloud gave me "Immediate connect fail for <ip6 of host>: Network unreachable"

on Nextcloud AIO v9.0.1

**Comment by szaimen** [comment]
Hi, please ask this at https://help.nextcloud.com. Thanks!

**Reply by isdnfan** [reply]
for reference https://help.nextcloud.com/t/ipv6-only-cant-connect-to-itself/196260

**Comment by szaimen** [comment, marked as answer]
The solution is in https://help.nextcloud.com/t/ipv6-only-cant-connect-to-itself/196260/5
